Libraries/LibGUI/TextEditor.cpp

```diff
@@ -1198,11 +1198,13 @@ void TextEditor::paste()
 {
     if (is_readonly())
         return;
-    auto paste_text = Clipboard::the().data();
-    printf("Paste: \"%s\"\n", paste_text.characters());
+    auto [data, type] = Clipboard::the().data_and_type();
+    if (type != "text")
+        return;
+    printf("Paste: \"%s\"\n", data.characters());
```

Comment:
```suggestion
    dbg() << "Paste: _" << data << "_";
```

Reply:
That line was already there wrote by @awesomekling so I didn't change it to don't over-do things for this PR. Shall I commit that change too? It seemed weird when I saw it, too.